packages/blockchain-common/src/registries/FabricWalletRegistry.ts

```diff
@@ -51,4 +54,49 @@ export class FabricWalletRegistry extends FileRegistry<FabricWalletRegistryEntry
 
         return entries;
     }
+
+    public async get(name: string, fromEnvironment?: string): Promise<FabricWalletRegistryEntry> {
+        const entries: FabricWalletRegistryEntry[] = await this.getAll();
+
+        const entry: FabricWalletRegistryEntry = entries.find((item: FabricWalletRegistryEntry) => {
+            if (item.fromEnvironment) {
+                return item.name === name && item.fromEnvironment === fromEnvironment;
+            } else {
+                return item.name === name;
+            }
+        });
+
+        if (!entry) {
+            throw new Error(`Entry "${name}" from environment "${fromEnvironment}" in registry "${FileConfigurations.FABRIC_WALLETS}" does not exist`);
```

Comment:
If `fromEnvironment` is undefined, then the message seems a bit odd.

```
if(!entry && fromEnvironment){
    throw new Error(`Entry "${name}" from environment "${fromEnvironment}" in registry "${FileConfigurations.FABRIC_WALLETS}" does not exist`);
} else if(!entry && !fromEnvironment){
    throw new Error(`Entry "${name}" in registry "${FileConfigurations.FABRIC_WALLETS}" does not exist`);
} else {
    return entry;
}
```

Or something else might be better!

Reply:
done